packages/graphback-runtime-mongodb/src/OffixMongoDBDataProvider.ts

```diff
@@ -42,6 +43,13 @@ export class OffixMongoDBDataProvider<Type = any, GraphbackContext = any> extend
       }
       // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
       data.version = data.version + 1;
+
+      this.fieldTransforms
+      .filter((t: FieldTransform) => t.transformType === TransformType.UPDATE)
+      .forEach((f: FieldTransform) => {
+        data[f.fieldName] = f.transform(f.fieldName);
+      });
+
       // TODO use findOneAndUpdate to check consistency afterwards
       const result = await this.db.collection(this.collectionName).updateOne({ _id: new ObjectId(idField.value) }, { $set: data });
       if (result.result?.ok) {
```

Comment:
Is it possible to re-use the `update` method we have in `MongoDBDataProvider.ts`?

Reply:
I believe so, yes, thanks!